libensemble/gen_funcs/persistent_deap_nsga2.py

```diff
@@ -68,7 +68,7 @@ def evaluate_pop(g, deap_object, Out, comm):
         Out['generation'][index] = g
     # Sending work to sim_f, which is defined in main call script
     # A fitness value will be returned in calc_in
-    tag, Work, calc_in = sendrecv_mgr_worker_msg(comm, Out)
+    tag, Work, calc_in = sendrecv_mgr_worker_msg(comm, Out[['individual', 'generation']])
```

Comment:
Do you need to specify these fields here? Last points will be false anyway?

Reply:
The reason is more to show/test that something such as `'last_points'` is never communicated from the persistent user script (only from the worker after the persistent worker has ended).